MAUI/PDF-Viewer/Localization.md

```diff
@@ -88,6 +88,12 @@ public partial class App : Application
 {% endhighlight %}
 {% endtabs %}
 
+N> When localizing multiple Syncfusion controls in a MAUI application, you should store all localization keys (name-value pairs) for the controls in a single resource (.resx) file. Then, set the `ResourceManager` using the unified resource file as shown below:
```

Comment:
There is no explanation provided what happens if we multiple resource files. 

<img width="535" height="117" alt="image" src="https://github.com/user-attachments/assets/52c20708-adbf-4f88-8e87-aefdf83e3906" />